canvas/models.py

```diff
@@ -214,8 +214,16 @@ def username(self):
         return self.user.username
 
     @property
-    def name(self):
-        return self.user.get_full_name()
+    def first_name(self):
```

Comment:
We don't need to expose first_name and last_name separately. And as I mentioned before email is an internal property we front-end only works with the username. Please remove the email.

Reply:
Fixed.